app/assets/javascripts/reducers/columns/backlog.js

```diff
@@ -50,13 +51,24 @@ const orderByState = stories => {
   ];
 };
 
+const groupStoriesInSprints = (stories, project) => {
+  const currentSprintNumber = Iteration.getCurrentIteration(project) || 0;
```

Comment:
Do you remember why we need this fallback?

Reply:
We use the `currentSprintNumber` as `initialSprintNumber` at `iteration` file. We use this number to calculate the sprints overflow.